lambda/main.go

```diff
@@ -69,6 +70,15 @@ func LambdaHandler(event NewRelease) error {
 		return err
 	}
 
+	isPre, err := isPreRelease(version)
+	if err != nil {
+		return err
+	}
+	if isPre {
+		log.Printf("Version %s is pre-release. Nothing to do. Exiting.", version)
```

Comment:
I read this as "exCITE-ing" and thought "but Josh doesn't like fun, what is going on, also, "nothing to do isn't exiting, it's boring"